src/core/server/models/tenant/tenant.ts

```diff
@@ -93,7 +100,7 @@ export interface TenantSettings
   /**
    * featureFlags is the set of flags enabled on this Tenant.
    */
-  featureFlags?: GQLFEATURE_FLAG[];
+  featureFlags?: (GQLFEATURE_FLAG | LEGACY_FEATURE_FLAGS)[];
```

Comment:
Maybe this might read cleaner?

```suggestion
  featureFlags?: Array<GQLFEATURE_FLAG | LEGACY_FEATURE_FLAGS>;
```